src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person | null,
+};
+
+export const PersonLink:React.FC<Props> = ({ person }) => {
+  const isName = person?.name || '-';
```

Comment:
This is not a boolean
```suggestion
  const name = person?.name || '-';
```